pokepaylib/src/main/java/jp/pokepay/pokepaylib/JsonConverter.java

```diff
@@ -22,10 +22,9 @@ public static ObjectMapper createObjectMapper() {
                 .configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
     }
 
-    public static String toString(final Map<String, Object> map)
-            throws JsonGenerationException, JsonMappingException, IOException {
-        final JSONObject j = new JSONObject(map);
-        return j.toString();
+    public static String toString(final Map<String, Object> map) throws IOException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        return objectMapper.writeValueAsString(map);
```

Comment:
May I know why this got changed?

Reply:
The purpose is to be able to run tests with JUnit without android simulator.